### ISSUE TITLE
Notification volume control error in android 12

### ISSUE THREAD

**Issue by drk002** [signalapp/Signal-Android#12126]
I have an issue with Signal notifications after updating to MIUI 13 Android 12
model: Poco X3 PRO

Notification sounds do not respond to the "sounds" setting slider https://github.com/signalapp/Signal-Android/issues/1 in the attached image. To increase or decrease the volume of notifications I have to use the global "multimedia" slider https://github.com/signalapp/Signal-Android/issues/2 in the picture.
The audio of the video call works fine. 

In MIUI 12.5 notifications worked very well you could set them with "sounds" slider.
I'll add that the ringtones in the system application "phone" respond to the slider "sounds" No. 1 and you can adjust them.
![1_proc](https://user-images.githubusercontent.com/65331799/161934490-b342a502-6c26-4318-bdb2-f1adfcc3c879.jpg)

**Comment by drk002** [comment]
My debuglog:
https://debuglogs.org/android/5.34.10/a4b0c140b27324779be0b8b8fbb18323a304f642d8b2087f4769e9a8087b1662

**Comment by drk002** [comment]
I read online that the notification system has been changed in MIUI 13 so it might not be a Signal issue.

**Comment by Xigglez** [comment]
Same issue on my Poco F2 Pro and friend's Xiaomi Mi 10 after updating to MIUI 13.0.2.0 (SJKEUXM) & Andriod 12 (SKQ1.211006.001). Notifications from other apps (including watsapp) does not have this problem.

**Comment by adilborekciler** [comment]
I have same problem too. I am using Redmi Note 10 Pro - MIUI 13.0.3.0 atm (latest rom for Turkey)
Signal notifications was working on MIUI 12.5 enchanced. 
I wrote on micommunity https://c.mi.com/thread-4023579-1-0.html
I contacted to signal support on 03.May.2022. I totaly sent 7 emails to them. But signal does not fixing the issue, Xiaomi neither. Mail screens in attachment.
![Screenshot from 2022-05-26 09-53-34](https://user-images.githubusercontent.com/12841934/170434369-e82a1c31-e790-445f-b662-fcda2c59df43.png)
I bought new phone for my wife last week. I bought Samsung with no doubt. I set every app on samsung. Wow, everything (even Signal App) is working as intended on samsung.

**Comment by lpirl** [comment]
Same on Lineage 18 and 19.

I also cannot mute notification sounds. My suspicion is that Signal ends up using the media playback for notifications somehow.

**Comment by pederbe** [comment]
I have the same issue. I'm on Xiaomi MIUI 13.0.4 and Signal 5.42.8.

This issue often leads to the Signal notification volume being unexpectedly loud (or sometimes too quiet).

**Comment by drk002** [comment]
Signal uses the multimedia settings for notifications for unknown reasons and that is why there is this issue. There is no such problem in other messengers.

**Comment by graue70** [comment]
I have the same problem. It started after upgrading from LineageOS 18 (based on Android 11) to LineageOS 19 (based on Android 12).
Other apps and even Signal calls don't have this problem. It's only the notifications.

**Comment by greyson-signal** [comment]
Are you experiencing this issue with notifications in the background, or only when the app is in the foreground?

**Comment by drk002** [comment]
Both in the background and in the foreground.

**Comment by CatalinFrancu** [comment]
Same here after reinstalling (i.e. reflashing, not upgrading) from LineageOS 17.1 to 19.1 (Android 10 to 12), on a Samsung Galaxy  S9.

**Comment by lnwor** [comment]
Same problem after updating to Lineage 19, I have an S10. It's really annoying

**Comment by pederbe** [comment]
Is there any more information needed from us to progress on this issue? 

Not sure if anyone is working on this problem because it's labelled "acknowledged" but no one is assigned yet.

**Comment by cody-signal** [comment]
We are tracking it internally.

**Comment by ManuVice** [comment]
same problem here on PocoF3 with MIUI 13 Android 12

**Comment by realwax** [comment]
Same issue with Poco F4 and latest updates.
Afaik the reason is that it uses the media volume instead of notification volume. Other app like Telegram, Whatsapp work, so it needs to be with Signal being compatible as other are...

**Comment by graue70** [comment]
I think this is solved for me. I fear I might be doing something wrong though. 😅
Can somebody please confirm or deny?

**Comment by adilborekciler** [comment]
It is not solved for me. I am using Redmi Note 10 Pro - MIUI 13.0.7.0 atm (latest rom for Turkey).
Notification sound is not fixed yet. But now I have an additional problem, notifications on screen are  completely gone. I have to open Signal App to check that I have any new messages..... This is ridiculous.

**Comment by realwax** [comment]
> It is not solved for me. I am using Redmi Note 10 Pro - MIUI 13.0.7.0 atm (latest rom for Turkey). Notification sound is not fixed yet. But now I have an additional problem, notifications on screen are completely gone. I have to open Signal App to check that I have any new messages..... This is ridiculous.

@adilborekciler did you reset your notifications? Try reconfigure to be sure. If custom ones are set they need to be reconfigured as described. If not, there should be Miui 14 for your phone soon...

**Comment by lpirl** [comment]
> I think this is solved for me. I fear I might be doing something wrong though. sweat_smile Can somebody please confirm or deny?

Yes, it somehow works for me also since a couple of weeks. Signal 6.11.7, Android 12, Lineage 19-20230202.

**Comment by KamilKurde** [comment]
It works for me on Xiaomi.eu 14.0.10.0.TLACNXM, but I have to go to conversation settings, notifications, and disable custom notifications for each thread I want to get sounds on

**Comment by adilborekciler** [comment]
> > It is not solved for me. I am using Redmi Note 10 Pro - MIUI 13.0.7.0 atm (latest rom for Turkey). Notification sound is not fixed yet. But now I have an additional problem, notifications on screen are completely gone. I have to open Signal App to check that I have any new messages..... This is ridiculous.
> 
> @adilborekciler did you reset your notifications? Try reconfigure to be sure. If custom ones are set they need to be reconfigured as described. If not, there should be Miui 14 for your phone soon...

ofc I did. But it's not fixed. So I will wait for MIUI14 then. I hope it will be fixed with 14
